components/country/OverviewCharts.js

```diff
@@ -31,24 +32,33 @@ const StyledTestGroupSelector = styled(Flex)`
   }
 `
 
-const StyledCallToActionBox = styled(Box)`
-  border: 2px solid ${props => props.theme.colors.blue5};
-`
-
 const CallToActionBox = () => {
   const { countryName } = useCountry()
   return (
-    <Flex my={4}>
-      <StyledCallToActionBox mx='auto' px={4}>
+    <Flex my={4} bg='gray3' flexWrap='wrap'>
+      <Box width={1} mx={4} my={2}>
+        <Heading h={4}>
+          <FormattedMessage id='Country.Overview.NoData.Title' />
+        </Heading>
         <Text fontSize={2}>
           <FormattedMarkdown
-            id='Country.Overview.CallToAction'
+            id='Country.Overview.NoData.CallToAction'
             values={{
               country: countryName
             }}
           />
         </Text>
-      </StyledCallToActionBox>
+      </Box>
+      <Flex alignItems='center' mx={4} my={4} flexDirection={['column', 'row']}>
+        <Box mr={4} mb={[3, 0]}>
+          <NLink href='https://ooni.org/install'><a>
+            <Button><FormattedMessage id='Country.Overview.NoData.Button.InstallProbe' /></Button>
+          </a></NLink>
+        </Box>
+        <Box>
+          <Button hollow><FormattedMessage id='Country.Overview.NoData.Button.OoniRunLink' /></Button>
```

Comment:
As I was mentioning on the channel, I think we should table the OONI Run link button for the moment and just go for the OONI Probe Install link.

Having an OONI Run link which just points to the country list, is effectively the same as having people tap on the websites card, so we should think more about what it is we want to support doing on this page.

Reply:
Dropped button for OONI Run